www/src/record/record.list.html

```diff
@@ -1,4 +1,4 @@
-<ion-view view-title="Registros">
+<ion-view view-title="{{ vm.research.name }}">
```

Comment:
Why not use the $ionicNavBarDelegate to set the title ? It's one less binding, or we can use one-way data binding with ```{{ ::vm.research.name }}```, what do you thing ?